adapters/vnfm_generic/polld/vnfm_generic_poll.php

```diff
@@ -80,7 +80,13 @@ function get_health($PROTOCOL, $ME_IP, $HTTP_PORT, $BASE_URL_MS, $TOKEN)
   {
     global $health_path;
 
-    $url = "$PROTOCOL://$ME_IP:$HTTP_PORT/$BASE_URL_MS/$health_path";
+    //$url = "$PROTOCOL://$ME_IP:$HTTP_PORT/$BASE_URL_MS/$health_path";
+     $url = "$PROTOCOL://$ME_IP:$HTTP_PORT";
+    if (!(empty($BASE_URL_MS)) && ($BASE_URL_MS !== '/')){
+        $url .= "/$BASE_URL_MS";
```

Comment:
Can you ensure that BASE_URL_MS does not start or end with the / character?

Reply:
@nku-ubiqube 
can you adjust the code based on @yguubiqube request ?

Reply:
Updated